scripts/cloud/fabfile.py

```diff
@@ -54,7 +54,7 @@ def installBasePackages():
         # Update image and install needed base components
```

Comment:
This seems pretty outdated. Worth keeping around?

Reply:
You are right, it is likely very outdated and I have never actually used or tested it. It might be handy as a EC2 reference, but I am sure there are also things that changed there over time. Therefore I think it is okay to remove it.

Reply:
Aaaaand its gone.